src/pages/Portfolio/LendingStats/LendingStatsPage.vue

```diff
@@ -135,24 +153,38 @@ export default {
 		preFetch: true,
 		result({ data }) {
 			const countriesWhereKivaDoesntWork = data?.general?.excluded_countries?.value ?? '';
+			/* eslint-disable max-len */
+			const activitiesWhereKivaDoesntWork = (data?.general?.excluded_activities?.value ?? '').split(',').map(item => item.replaceAll('"', '')).map(item => convertToNumberOrString(item));
```

Comment:
Why is the `replaceAll` needed for these but not `excluded_countries`?

Reply:
For countries we are not splitting based on a comma, said that, as a result we have an array including `"` in elements we need to clean to match with data.

We are doing this to have support to use names or IDs, for countries we use ISO codes, so we don't need same approach.